zutil.h

```diff
@@ -38,10 +38,10 @@ typedef unsigned char uch; /* Included for compatibility with external code only
 typedef uint16_t ush;      /* Included for compatibility with external code only */
 typedef unsigned long ulg;
 
-extern const char * const zng_errmsg[10]; /* indexed by 2-zlib_error */
+extern const char * const PREFIX(z_errmsg)[10]; /* indexed by 2-zlib_error */
 /* (size given to avoid silly warnings with Visual C++) */
 
-#define ERR_MSG(err) zng_errmsg[Z_NEED_DICT-(err)]
+#define ERR_MSG(err) PREFIX(z_errmsg)[Z_NEED_DICT-(err)]
```

Comment:
Since now using `PREFIX` inside zutil.h, we have to always include zbuild.h before including zutil.h in any source files.

Reply:
@Dead2 Any thoughts on this change?

Reply:
It looks OK. Though I was also wondering whether it might be just as well to include zbuild.h from zutil.h.

Reply:
I have included `zbuild.h`.